vmdk_plugin/utils/refcount/refcnt.go

```diff
@@ -267,6 +264,19 @@ func (r *RefCountsMap) GetCount(vol string) uint {
 	return rc.count
 }
 
+//GetVolumeNames - return volume names of entries in refmap
+func (r *RefCountsMap) GetVolumeNames() []string {
+	r.mtx.RLock()
+	defer r.mtx.RUnlock()
+	var volumeList []string
+
+	for k := range r.refMap {
+		volumeList = append(volumeList, k)
+	}
+
+	return volumeList
+}
```

Comment:
Sure. did you mean "list can contain volumes with same name on different datastores?"